### PR TITLE
fix: link target to new tab for in-app notifications

### PR DESCRIPTION
**Pull request by DorisWongNHS**
To be work with FE change: https://github.com/Health-Education-England/tis-trainee-ui/pull/1034
This PR need to go first

<NO_TICKET>

### CODE REVIEW

File: src/main/resources/templates/in-app/sponsorship/v1.0.0.html [hunk 1]
```diff
@@ -21,7 +21,12 @@ <h2>Content</h2>
         >.
       </p>
       <p>
-        If you require a Certificate of Sponsorship in order to apply for a Skilled Worker visa for this rotation, please follow this <a href="https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/overseas-applicants/overseas-sponsorship-guidance/overseas-sponsorship-contact-details">link</a> for details on the application process. Should you have any questions that are not answered on this website,
+        If you require a Certificate of Sponsorship in order to apply for a Skilled Worker visa for this rotation, please follow this
+        <a
+          href="https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/overseas-applicants/overseas-sponsorship-guidance/overseas-sponsorship-contact-details"
+          th:target="_blank"
+        >link</a>
```
Comment (john12321) [inline src/main/resources/templates/in-app/sponsorship/v1.0.0.html:28]: @DorisWongNHS , in true JFDI spirit, we could make this link text more descriptive e.g. instead of 'link' we could do 'link to the overseas sponsorship application process contact details'. It's still a bit wordy but at least the user should have more idea what they are clicking 😁 (ignore me if it's part of a follow-up ticket etc.)

File: src/main/resources/templates/in-app/sponsorship/v1.0.0.html [hunk 2]
```diff
@@ -31,6 +36,7 @@ <h2>Content</h2>
             <a
```
Comment (john12321) [inline src/main/resources/templates/in-app/sponsorship/v1.0.0.html:36]: Similar to above, we can probably add some descriptive text and then just hyperlink it rather than use the raw https link? 

e.g. "link to medical specialty training resources for overseas applicants"

Comment (DorisWongNHS) [reply src/main/resources/templates/in-app/sponsorship/v1.0.0.html:36]: Yeah Thanks @john12321 , I notice that for Deferral, LTFT and Sponsorship notification as well. The text may differ according to email or href contact type, and the notification type as well. So that will need to confirm with Naz and in separate PR.

**Review by john12321 — DISMISSED** [review]

**Review by Judge40 — APPROVED** [review]
